app/uk/gov/hmrc/cataloguefrontend/platforminitiatives/PlatformInitiativesListPage.scala.html

```diff
@@ -61,7 +73,10 @@ <h1>Platform Initiatives@if(team!=""){: @team}</h1>
         <div class="right padding-reset-right" style="line-height:2.2em;">
             @Seq(DisplayType.Chart, DisplayType.Progress).zipWithIndex.map { case (dt, i) =>
                 @if(i != 0){ | }
-                <a href="@appRoutes.PlatformInitiativesController.platformInitiatives(dt)">
+                @team match {
+                    case ""      => { <a href="@appRoutes.PlatformInitiativesController.platformInitiatives(dt)"> }
+                    case _       => { <a href="@appRoutes.PlatformInitiativesController.platformInitiatives(dt)&team=@team"> }
```

Comment:
is the team not a parameter of `platformInitiatives`? We should try and avoid adding params to the resultant String. 
Can we not model `team` as an `Option[String]` rather than an empty String? Then we could simply call `@appRoutes.PlatformInitiativesController.platformInitiatives(dt, optTeam)`